keyboards/levinson/keymaps/jyh/keymap.c

```diff
@@ -0,0 +1,232 @@
+ * |------+------+------+------+------+------|    |------+------+------+------+------+------|
+ * |      | Left |  Up  | Down | Right|  (   |    |  )   |  -   |  =   |  [   |  ]   |  |   |
+ * |------+------+------+------+------+------|    |------+------+------+------+------+------|
+ * |      |  ;   |  '   |  *   |  ,   |  {   |    |  }   |  _   |  +   |  .   |  /   |      |
+ * |------+------+------+------+------+------|    |------+------+------+------+------+------|
+ * |      |      |      |      |      |      |    |      |      |      |      |      |      |
+ * `-----------------------------------------'    `-----------------------------------------'
+ */
+				[_RAISE] = KC_KEYMAP(
+								HPR_GRV, 1  , 2  , 3  , 4  , 5  ,      6  , 7  , 8  , 9  , 0  ,       ,
+               ,LEFT, UP ,DOWN,RGHT,LPRN,     RPRN,MINS, EQL,LBRC,RBRC,CTL_PIP,
+								       ,COLN,QUOT,ASTR,COMM,LCBR,     RCBR,UNDS,PLUS,    ,    ,       ,
+								       ,    ,    ,    ,    ,    ,         ,    ,    ,    ,    ,
+								),
+
+/* Lower
+ * ,-----------------------------------------.    ,-----------------------------------------.
+ * |  ~   |  !   |  @   |  #   |  $   |  %   |    |  ^   |  &   |  *   |  (   |  )   |      |
+ * |------+------+------+------+------+------|    |------+------+------+------+------+------|
+ * | Del  | Vol+ | Mute | Play | PgUp |  :   |    |  '   |  -   |  =   |  [   |  ]   |  \   |
+ * |------+------+------+------+------+------|    |------+------+------+------+------+------|
+ * |      | Vol- | < M  |  M > | PgDn |  ;   |    |  "   |  |   |  ,   |  .   |  /   |      |
+ * |------+------+------+------+------+------|    |------+------+------+------+------+------|
+ * |      |      |      |      |      |      |    |      |      |      |      |      |      |
+ * `-----------------------------------------'    `-----------------------------------------'
+ */
+				[_LOWER] = KC_KEYMAP(
+								HPR_TIL,EXLM, AT ,HASH,DLR , PERC  ,     CIRC,AMPR,ASTR,LPRN,RPRN,       ,
+								CTL_DEL,VOLU,MUTE,MPLY,PGUP, COLN  ,     QUOT,MINS, EQL,LBRC,RBRC,CTL_BSL,
+								       ,VOLD,MPRV,MNXT,PGDN, SCLN  ,     DQUO,PIPE,COMM, DOT,SLSH,       ,
+								       ,    ,    ,    ,    ,       ,         ,    ,    ,    ,    ,
+								),
+
+				[_FUNC] = KC_KEYMAP(
+// ,----+----+----+----+----+----.    ,----+----+----+----+----+----.
+								, F1 , F2 , F3 , F4 , F5 ,      F6 , F7 , F8 , F9 , F10,    ,
+// |----+----+----+----+----+----|    |----+----+----+----+----+----|
+								, F11, F12, F13, F14, F15,      F16, F17, F18, F19, F20,    ,
+// |----+----+----+----+----+----|    |----+----+----+----+----+----|
+								, F21, F22, F23, F24, NO ,      NO , NO , NO , NO , NO ,    ,
+// |----+----+----+----+----+----|    |----+----+----+----+----+----|
+								,    ,    ,    ,    ,    ,         ,    ,    ,    ,    ,
+// `----+----+----+----+----+----'    `----+----+----+----+----+----'
+  ),
+
+				[_NUMS] = KC_KEYMAP(
```

Comment:
`KC_KEYMAP` is no longer used for this keyboard

Please replace this with `LAYOUT_kc` or add KC_ to all of the keys and use  `LAYOUT_ortho_4x12`

```suggestion
				[_NUMS] = LAYOUT_kc(
```

```suggestion
				[_NUMS] = LAYOUT_ortho_4x12(
```